packages/surveys/src/types.ts

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { IItem } from "@esri/arcgis-rest-types";
+
+type IResultsAvailability = "after" | "always";
+
+export interface IFormItemProperties {
+  [propName: string]: any;
+  previewUrl?: string;
+  parentId?: string;
+  settings?: {
+    resultsAvailability: IResultsAvailability;
+  };
+}
+
+export interface IFormItem extends IItem {
```

Comment:
I'm a little concerned about our interchangeable use of `Survey` and `Form`. I understand that the AGO Item type is `Form`. However from Hub domain model we refer to `survey`. Would it make a better DevEx to use Survey consistently?

Reply:
That's a fair point. I think (and @rweber-esri can correct me if I'm wrong) in `opendata-ui` we tend to refer to `survey` as the composite collection of related things (form item, form.json, views). Worth a discussion nevertheless.

Reply:
@cpgruber that's correct. There's a few places where we interact with more than just the `Form` item, and I've been using `Survey` as a higher level aggregate type comprised of the different survey item models.